### PR TITLE
Updating dashboard and adding new gadgets

### PR DESCRIPTION
**Pull request by ksdperera**
Adding commons  and  LogAPIMMessageProcessing gadgets.
Updating dashboard and other gadgets.

### CODE REVIEW

File: log-analyzer/features/org.wso2.carbon.la.apim.feature/src/main/resources/gadgets/LogAPIMMessageProcessing/js/VizGrammar.js [hunk 1]
```diff
@@ -0,0 +1,2300 @@
+/*
```
Comment (keizer619) [inline log-analyzer/features/org.wso2.carbon.la.apim.feature/src/main/resources/gadgets/LogAPIMMessageProcessing/js/VizGrammar.js:1]: @ksdperera  Please use minified JS

File: log-analyzer/features/org.wso2.carbon.la.apim.feature/src/main/resources/gadgets/commons/js/gadget-utils.js [hunk 2]
```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (keizer619) [inline log-analyzer/features/org.wso2.carbon.la.apim.feature/src/main/resources/gadgets/commons/js/gadget-utils.js:2]: @ksdperera  update license header year

File: log-analyzer/features/org.wso2.carbon.la.apim.feature/src/main/resources/gadgets/LogAPIMMessageProcessing/js/VizGrammar.js (hunk 1 above)
Comment (ksdperera) [reply log-analyzer/features/org.wso2.carbon.la.apim.feature/src/main/resources/gadgets/LogAPIMMessageProcessing/js/VizGrammar.js:1]: Hi Thanrik,

This is fixed by the commits in : https://github.com/wso2/shared-analytics/pull/126

Thanks,
SajithD

File: log-analyzer/features/org.wso2.carbon.la.apim.feature/src/main/resources/gadgets/commons/js/gadget-utils.js (hunk 2 above)
Comment (ksdperera) [reply log-analyzer/features/org.wso2.carbon.la.apim.feature/src/main/resources/gadgets/commons/js/gadget-utils.js:2]: Hi Thanrik,

This is fixed by the commits in : https://github.com/wso2/shared-analytics/pull/55

Thanks,
SajithD

**Pull request merged**